crmsh/bootstrap.py

```diff
@@ -414,6 +414,10 @@ def check_prereqs(stage):
             warn("No watchdog device found. If SBD is used, the cluster will be unable to start without a watchdog.")
             warned = True
 
+    if stage == "qdevice":
+        if not _context.qdevice:
+            error("Miss qdevice related option(at least with --qdevice)")
```

Comment:
I would change this to:
`qdevice related options are missing (--qdevice option is mandatory, find for more information using --help)`

Reply:
Changed,
thanks!